apidoc/Titanium/UI/iOS/Toolbar.yml

```diff
@@ -33,12 +33,15 @@ description: |
     *  [Toolbar in the iOS Human Interface Guidelines.](https://developer.apple.com/library/ios/documentation/userexperience/conceptual/mobilehig/Bars.html#//apple_ref/doc/uid/TP40006556-CH12-SW4)
 
 extends: Titanium.UI.View
-excludes: { properties: [ backgroundColor, backgroundDisabledColor, backgroundDisabledImage,         
+excludes:
+    properties: [backgroundColor, backgroundDisabledColor, backgroundDisabledImage,         
             backgroundFocusedColor, backgroundFocusedImage,
             backgroundLeftCap, backgroundTopCap, backgroundRepeat,
             backgroundSelectedColor, backgroundSelectedImage, layout,
-            softKeyboardOnFocus ],
```

Comment:
I think we need to drop the comma on this line.

Reply:
It is a removed line, the new line is 41 (`softKeyboardOnFocus]`)